hazelcast/src/main/java/com/hazelcast/cluster/impl/JoinMessage.java

```diff
@@ -34,25 +36,25 @@
     protected String uuid;
     protected boolean liteMember;
     protected ConfigCheck configCheck;
-    protected Collection<Address> memberAddresses;
+    protected Collection<Member> members;
```

Comment:
I used them to count data members on the merging side but I think we can only send number of data members